.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  root: true,
+  env: {
+    node: true
+  },
+  'extends': [
+    'plugin:vue/essential',
```

Comment:
please change `essential` rule to `recommended`.
https://eslint.vuejs.org/rules/

Reply:
Changed, my bad